iib/workers/tasks/opm_operations.py

```diff
@@ -295,196 +295,132 @@ def inner(*args, **kwargs):
     return decorator
 
 
-def opm_serve_from_index(base_dir: str, from_index: str) -> Tuple[int, subprocess.Popen]:
+def get_operator_package_list(
+    input_data: str,
```

Comment:
input_data seems very generic and doesn't show any meaning, how about we name it something like input_image_or_path